src/GeneralStateTestsFiller/stCreateTest/CREATE_empty000CreateinInitCode_TransactionFiller.json

```diff
@@ -96,7 +96,7 @@
             }
         },
         "transaction" : {
-	    "data" : "{(CALL 60000 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 0) (CREATE 0 0 0) }",
+	    "// data" : "{(CALL 60000 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b 0 0 0 0 0) (CREATE 0 0 0) }",
```

Comment:
Shouldn't all these LLL cases remain as LLL source code?

Reply:
I think so, I created an issue about it https://github.com/ethereum/tests/issues/433